ui/src/app/customizations/api-connectors/api-connector-list.component.scss

```diff
@@ -0,0 +1,26 @@
+@import 'patternfly-sass/assets/stylesheets/patternfly/color-variables';
+
+:host {
+  flex: 1 1 auto;
+  height: 100%;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  padding-bottom: 20px;
+  // Dock the scrollbar to the right
+  margin-right: -20px;
+  padding-right: 20px;
+}
+
+.api-connector-list {
+  height: 100%;
+
+  .col-xs-12 {
+    height: 100%;
+  }
+
+  & /deep/ .toolbar-pf {
```

Comment:
uh.. .What out for this in the future ;) https://github.com/syndesisio/syndesis/issues/366

Reply:
Copy pasta! What can I say..